config/api.yml

```diff
@@ -477,10 +477,17 @@
       :post:
       - :name: query
         :identifier: cloud_volume_show_list
+      - :name: delete
```

Comment:
The `resource_actions` `delete` which is under `post` is covered by the test.
And that would be the test for it https://github.com/ManageIQ/manageiq/pull/15097/files#diff-3eaddeceef284aecbf30c2f57658d722R52